src/TimePicker/TimePickerDialog.spec.js

```diff
@@ -0,0 +1,29 @@
+// @flow weak
+/* eslint-env mocha */
+import React from 'react';
+import { createShallowWithContext } from 'test/utils';
+import { expect } from 'chai';
```

Comment:
We don't use any expect. Please use the `assert` API that is much simpler to learn and to use.